src/rust/demikernel/bindings.rs

```diff
@@ -512,10 +512,28 @@ pub extern "C" fn demi_getsockopt(
 // Standalone Functions
 //==============================================================================
 
-/// Converts a [sockaddr] into a [Ipv4Endpoint].
+/// Converts a [sockaddr] into a [SocketAddrV4].
 fn sockaddr_to_ipv4endpoint(saddr: *const sockaddr) -> Result<SocketAddrV4, Fail> {
```

Comment:
This function name is left over from when we still had an IPv4Endpoint structure.  Can you change it to something like sockaddr_to_SocketAddrV4?

Reply:
Or if you want to start putting in multi-protocol support, make it sockaddr_to_SocketAddr (and return a SocketAddr).

Reply:
Sure. I renamed this function.

Reply:
Thanks!